lib/runner.js

```diff
@@ -130,11 +158,45 @@ class Runner {
         process.send({
             event: 'runner:end',
             failures: failures,
-            cid: this.cid
+            cid: this.cid,
+            specs: this.specs
         })
         process.exit(failures === 0 ? 0 : 1)
     }
 
+    addTestDetails (payload) {
+        if (this.currentTest) {
+            payload.title = this.currentTest.title
+            payload.parent = this.currentTest.parent
```

Comment:
We need to solve this differently. I am sure that Jasmine doesn't provide you title and parent that easily (also in Cucumber this isn't quite applicable). Having that in the main repository seems not correct.